pom.xml

```diff
@@ -4,17 +4,17 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.2</version>
+    <version>1.560</version>
   </parent>
 
   <artifactId>promoted-builds</artifactId>
-  <version>2.24.2-SNAPSHOT</version>
+  <version>2.25-SNAPSHOT</version>
```

Comment:
No need to change it in the PR. It will cause merge conflicts if there is a hotfix going ahead your PR.